db/db_impl/db_impl.cc

```diff
@@ -562,6 +562,19 @@ Status DBImpl::CloseHelper() {
     delete txn_entry.second;
   }
 
+  std::unordered_set<std::string> db_paths;
```

Comment:
My 2 cents are to use std::set<> rather than unordered set. Most likely there is only one DB path.

Reply:
good point, the cost of hashing may not be worth it.